pkg/kubelet/container/helpers.go

```diff
@@ -91,7 +91,7 @@ func ShouldContainerBeRestarted(container *v1.Container, pod *v1.Pod, podStatus
 // HashContainer returns the hash of the container. It is used to compare
 // the running container with its desired spec.
 func HashContainer(container *v1.Container) uint64 {
-	hash := adler32.New()
+	hash := fnv.New32a()
```

Comment:
IIUC, all containers will be restarted after kubelet live upgrade because of this, I'm not sure whether this is acceptable. At least we should document this.

Luckily we are going to rollout CRI this release, which also requires restarting container. :p 

@yujuhong /cc @kubernetes/sig-node-pr-reviews